package.json

```diff
@@ -46,7 +46,7 @@
     "ember-cli-notifications": "^4.1.6",
     "ember-cli-test-loader": "^1.1.1",
     "ember-cli-uglify": "^1.2.0",
-    "ember-code-snippet": "^1.8.0",
+    "ember-code-snippet": "ciena-blueplanet/ember-code-snippet#remove-browserify",
```

Comment:
I did raise an [issue](https://github.com/ef4/ember-code-snippet/issues/34) with `ember-code-snippet` to make them aware of the issue, but in the meantime, we can just use our fork.